### PR TITLE
added links, more information and structure to the 3.2. release notes

### PR DESCRIPTION
**Pull request by joemoe**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 1]
```diff
@@ -86,114 +87,165 @@ Breaking Changes
 Changes
 -------
 
-- ``geo_shape`` columns can now be casted to ``object`` with ``cast`` in
-  addition to ``try_cast``.
+New features
```
Comment (smakalias) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]:
```suggestion
New Features
```

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 2]
```diff
@@ -86,114 +87,165 @@ Breaking Changes
...
+  `trial <enterprise_trial>`_ the
+  enterprise features, set a production enterprise license or continue
+  using the community edition. Additionally, a new 
+  `SET LICENSE <ref-set-license>`_ statement
+  has been added for license registration, and the ``license.ident`` setting
+  has become ``@deprecated``.
 
-- Added support for changing the number of shards on an existing table or
-  partition using the ``ALTER TABLE SET`` statement.
+SQL improvements
```
Comment (smakalias) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]:
```suggestion
SQL Improvements
```

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 3]
```diff
@@ -86,114 +87,165 @@ Breaking Changes
...
-   - Updated the license div to be responsive in the Admin UI.
+- Values of byte-size and time based configuration setting do not require a unit
+  suffix anymore. Without a unit time values are treat as milliseconds since
+  epoch and byte size values are treat as bytes.
+
+- Added support of using units inside byte-size or time bases statement
+  parameters values. E.g. '1mb' for 1 MegaByte or '1s' for 1 Second.
 
-   - Various Admin UI improvements.
+PostgreSQL compatibility
```
Comment (smakalias) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]:
```suggestion
PostgreSQL Compatibility
```

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 4]
```diff
@@ -86,114 +87,165 @@ Breaking Changes
...
+- Expand the `search_path <conf-session-search-path>`_ setting to 
+  accept a list of schemas that will be
+  searched when a relation (table, view or user defined function) is referenced
+  without specifying a schema. The system 
+  `pg_catalog <postgres_pg_catalog>`_ schema is implicitly
+  included as the first one in the path.
 
-- Upgraded to Elasticsearch 6.5.1, which includes changes to the default logging
-  configuration.
+Database administration
```
Comment (smakalias) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]:
```suggestion
Database Administration
```

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 5]
```diff
@@ -86,114 +87,165 @@ Breaking Changes
...
+  failures during operations which create such temporary artifacts.
 
-- Added the ``GENERATE_SERIES(start, stop [, step ])`` table function.
+- Added support for per-table 
+  `shard allocation filtering <ddl_shard_allocation>`_.
 
-- Added ``pg_class``, ``pg_namespace``, ``pg_attribute``, ``pg_attrdef``,
-  ``pg_index`` and ``pg_constraint`` tables to the ``pg_catalog`` schema for
-  improved compatibility with postgresql.
+Admin UI upgrade to 1.11.3
```
Comment (smakalias) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]:
```suggestion
Admin UI Upgrade to 1.11.3
```

**Comment by andreidan** [comment]
This is super useful. It's so much more information and it actually makes the release notes navigable. Well done @joemoe

**Comment by joemoe** [comment]
@smakalias i fixed your comments.

**Comment by seut** [comment]
retest this please

**Pull request merged**